README.md

```diff
@@ -0,0 +1,3 @@
+Instructions for upgrading custom error pages after modifications can
+be found in DIAS internal wiki, page "DIAS tech operations -
+instructions" → "Custom Error Pages".
```

Comment:
Voisiko tähän lisätä vielä esim
```suggestion
instructions" → "Custom Error Pages".

**Note:** this must be a public Github repository so that it can be server via Github Pages. Make sure not to push any non-public information in this repository!
```